app/colorbox.cpp

```diff
@@ -69,7 +69,7 @@ QColor ColorBox::color()
 
 void ColorBox::setColor(const QColor& newColor)
 {
-    if ( newColor != mColorWheel->color() )
+    if ( newColor.toHsv() != mColorWheel->color() )
```

Comment:
Just a question. What is the difference between of this colour comparison with and without `toHsv()`?

Reply:
When you choose RGB in the Pencil2D UI, newColor will be based on RGB colorspace while mColorWheel will always be in HSV colorspace. This means that it will always be false whenever you compare against two different color spaces. Therefore to avoid that I've added the .ToHsv, so we can be sure that it's always checking correctly.